source/usr/lib/mcvirt/iso/iso.py

```diff
@@ -0,0 +1,113 @@
+"""Provide class for managing ISO files"""
+
+# Copyright (c) 2015 - I.T. Dev Ltd
+#
+# This file is part of MCVirt.
+#
+# MCVirt is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MCVirt is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MCVirt.  If not, see <http://www.gnu.org/licenses/>
+
+import os
+import stat
+
+from mcvirt.exceptions import (InvalidISOPathException, NameNotSpecifiedException,
+                               IsoAlreadyExistsException, FailedToRemoveFileException,
+                               IsoInUseException)
+from mcvirt.utils import get_hostname
+from mcvirt.rpc.pyro_object import PyroObject
+from mcvirt.constants import DirectoryLocation
+
+
+class Iso(PyroObject):
+    """Provides management of ISOs for use in MCVirt"""
+
+    def __init__(self, name):
+        """Ensure the VM exists, checks the file permissions and creates
+        an Iso object.
+        """
```

Comment:
Just a comment issue: the docstring says ``Ensure the VM exists, checks the file permissions`` but I can't see anything to do with a VM here, and the file permissions are being set, not checked